datanode/gateway/graphql/schema.graphql

```diff
@@ -3759,8 +3759,8 @@ type EthCallSpec {
   Address: String!
   Abi: [String!]
   Method: String!
-  Args: [String!]
-  Trigger: EthCallTrigger!
+  Args: [String]
```

Comment:
In protobuf land args is a required list of [google.protobuf.Value](https://protobuf.dev/reference/csharp/api-docs/class/google/protobuf/well-known-types/value#:~:text=Protobuf.-,WellKnownTypes.,or%20a%20list%20of%20values.)

and while there is a `[NullValue]` in that `oneof` if we are simplifiying this in graphql to just the JSON representation of that value, it should be the string 'nil' not an actual null.

The trigger is definitely not optional either so we need to figure out how that is getting to be a null rather than allowing it in the schema

Reply:
Just to confirm @pscott31 you are saying  as I do, that this changed needs to be reverted, correct?

Reply:
yep